### ISSUE TITLE
Why bq client create dataset in data loader?

### ISSUE THREAD

**Issue by jx2lee** [mage-ai/mage-ai#5197]
I understood `data loader` is for fetching remote data or local files.

`__write_table` in the Bigquery export.__process function, which contains the line [client creates](https://github.com/mage-ai/mage-ai/blob/2da80d2216b87ee4a47875687f7f5f60ded0ef24/mage_ai/io/bigquery.py#L374) the dataset. If `data loader` use to export an existing dataset, I think it can leave out to create the dataset. (I haven't seen any warehouse IO that creates a bigquery dataset level resource. If I missed it, please share the link)

To summarize,
- **bigquery:[L374](https://github.com/mage-ai/mage-ai/blob/2da80d2216b87ee4a47875687f7f5f60ded0ef24/mage_ai/io/bigquery.py#L374)** can be deleted if there's a reason not to? Also, I haven't found any similar issues. If exists a reason why bq client should create a dataset, please let me know 🙏.
- If you accept this, I'd like to contribute. Can I create a PR myself (if the syntax is not needed)?

**Comment by jx2lee** [comment]
i found warehouse used `create_?` (? -> same level in bigquery.dataset)!
in redshift, create_schema parameter is existed. (default true)

Why not use the create_dataset parameter in bigquery.export?

**Comment by wangxiaoyou1993** [comment]
You can add the create_dataset param to the bigquery.export method. Feel free to create a PR for it.

**Comment by jx2lee** [comment]
@wangxiaoyou1993 Could you review the #5124 my colleague created to fix issue?
